uni/pubspec.yaml

```diff
@@ -103,28 +105,29 @@ flutter:
     - assets/images/
     - assets/env/
     - assets/text/
+    - assets/text/locations/
 
   # An image asset can refer to one or more resolution-specific "variants", see
   # https://flutter.dev/assets-and-images/#resolution-aware
 
   # For details regarding adding assets from package dependencies, see
   # https://flutter.dev/assets-and-images/#from-packages
 
-  # To add custom fonts to your application, add a fonts section here,
+  # To add custom Fonts to your application, add a Fonts section here,
   # in this "flutter" section. Each entry in this list should have a
-  # "family" key with the font family name, and a "fonts" key with a
+  # "family" key with the font family name, and a "Fonts" key with a
   # list giving the asset and other descriptors for the font. For
   # example:
-  # fonts:
+  # Fonts:
   #   - family: Schyler
-  #     fonts:
-  #       - asset: fonts/Schyler-Regular.ttf
-  #       - asset: fonts/Schyler-Italic.ttf
+  #     Fonts:
+  #       - asset: Fonts/Schyler-Regular.ttf
+  #       - asset: Fonts/Schyler-Italic.ttf
   #         style: italic
   #   - family: Trajan Pro
-  #     fonts:
-  #       - asset: fonts/TrajanPro.ttf
-  #       - asset: fonts/TrajanPro_Bold.ttf
+  #     Fonts:
+  #       - asset: Fonts/TrajanPro.ttf
+  #       - asset: Fonts/TrajanPro_Bold.ttf
```

Comment:
Review "Fonts" text here please

Reply:
Don't know how it got changed, but it's just comments. Changed it anyway